src/pages/zh-cn/core-concepts/project-structure.mdx

```diff
@@ -3,6 +3,7 @@ layout: ~/layouts/MainLayout.astro
 title: 项目结构
 description: 学习如何用 Astro 构建一个项目。
 ---
+import FileTree from '~/components/FileTree.astro'
 
 你用 `create-astro` CLI 向导所生成的新 Astro 项目中已经包括一些文件和文件夹。其他的则你将自己创建并添加到 Astro 的现有文件结构中。
```

Comment:
We recently updated this to remove the hyphen in `create astro`:

```suggestion
你用 `create astro` CLI 向导所生成的新 Astro 项目中已经包括一些文件和文件夹。其他的则你将自己创建并添加到 Astro 的现有文件结构中。
```

See: https://github.com/withastro/docs/commit/7abc2abe98077d7596ebbd38b979b62c5d20ba4b#diff-2f438cd874e276d48f0738cb2eae905c3407ec81207c7293a1bbaa13991bcea3R9